c3pzero_base_config/config/base_config.yaml

```diff
@@ -0,0 +1,28 @@
+###############################################################
+#
+#  This configures the robot to work with moveit studio
+#
+###############################################################
+
+hardware:
+  # Used by the ur_description package to set kinematics and geometry for a specific robot type.
+  type: "c3pzero"
```

Comment:
I think you can remove the `type` key and value here.They're used by the UR5e base config because its URDF has a particular parameter to change the robot type, but I don't think we have the same thing going on here.